bower.json

```diff
@@ -22,7 +21,7 @@
   ],
   "license": "Apache License, v2.0",
   "dependencies": {
-    "d3": "^3.3.13"
+    "d3": "~3.3.13"
```

Comment:
This is fix is necessary to get the unit tests working without code changes.  The interactiveBisect tests fail with v3.5 of D3.  With "`^`", `bower install` pulls down 3.5.3, I believe, where as "`~`" allows bower to install 3.3.13.

We'll just have to fix the interactiveBisect behavior a different way.